Tools/jit/_stencils.py

```diff
@@ -244,8 +252,9 @@ def remove_jump(self, *, alignment: int = 1) -> None:
                 jump = b"\x00\x00\x00\x14"
             case _:
                 return
-        if self.body[offset:] == jump and offset % alignment == 0:
+        if self.body[offset:] == jump:
             self.body = self.body[:offset]
+            self.disassembly = self.disassembly[:-2]
```

Comment:
Eh, I'd leave the disassembly alone. I get what you're doing here, but I'd rather leave it as what the preprocessed stencil used to look like and not try to do anything too clever.